### PR TITLE
sshguard: version bump to 2.3.1

### PR DESCRIPTION
**Pull request by ssiloti**

### CODE REVIEW

File: packages/net-security/sshguard/sshguard.exlib [hunk 1]
```diff
@@ -15,31 +15,23 @@ LICENCES="
...
-    'providers:hosts --with-firewall=hosts'
-    'providers:iptables --with-firewall=iptables'
-    'providers:iptables --with-iptables=/usr/host/bin/iptables'
-)
-
 sshguard_src_install() {
     default
     install_systemd_files
+
+    insinto /etc
```
Comment (sardemff7) [inline packages/net-security/sshguard/sshguard.exlib]: I know there are probably existing examples, but I’d rather install the sample file to `/usr/share/sshguard` to keep `/etc` as clean as possible.

File: packages/net-security/sshguard/sshguard.exlib [hunk 2]
```diff
@@ -15,31 +15,23 @@ LICENCES="
...
+MYOPTIONS=""
 
 DEPENDENCIES="
     build:
         sys-devel/bison
         sys-devel/flex
-    build+run:
-        providers:iptables? ( net-firewall/iptables )
+    recommendation:
+        net-firewall/iptables
```
Comment (sardemff7) [inline packages/net-security/sshguard/sshguard.exlib]: I take it it’s now automagic at runtime?

Comment (ssiloti) [reply packages/net-security/sshguard/sshguard.exlib]: Not really automagic, it all depends on which backend is selected in `sshguard.conf`. `sshg-fw-iptables` depends on `net-firewall/iptables`, `sshg-fw-nft-sets` depends on `net-firewall/nftables`, etc.

Comment (sardemff7) [reply packages/net-security/sshguard/sshguard.exlib]: I see, hence the `ls` in the news item, good.

**Comment by sardemff7** [comment]
Thanks again :-)

**Pull request merged**